### ISSUE TITLE
Regarding its application in the stock domain

### ISSUE THREAD

**Issue by shepherd0221** [Y-debug-sys/Diffusion-TS#133]
Hello,

I conducted a prediction experiment using S&P 500 stock OHLCV data.
In the YAML configuration, I only modified the seq_length, window, and the train_dataset proportion (changed to 0.9).
I used the commands from your GitHub repository to perform training and sampling. During the 1000 sampling runs, I specified different random seeds for each run.

The results are as follows:

<img width="1204" height="245" alt="Image" src="https://github.com/user-attachments/assets/3c742013-1a18-4d66-bfab-b90b25163b9c" />

The predicted data trends all look quite similar, and there is a noticeable deviation from the ground truth trend.
I’d like to confirm whether this is a normal phenomenon, or if there’s something I might have overlooked. Thank you.

**Comment by Y-debug-sys** [comment]
Hi, it might be due to overfitting. The model could be memorizing training patterns instead of generalizing. Try setting a non-zero dropout value (e.g., 0.1–0.3) in the configuration and retrain to see if the predictions diversify. Thanks!
